testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -801,13 +801,13 @@ def test_time_with_offset(self):
 
         assert_equal(ts.time, reftime + ref_offset)
 
-    def test_dt(self):
+    def test_dt(self, ref):
```

Comment:
Oddly enough the tests still fail. But if I directly change the files in where MD is installed (site-packages) then the same change works. Is there another build step I'm missing?!